app/helpers/events_helper.rb

```diff
@@ -110,4 +110,16 @@ def order_ga_accounts_for_employer_xml(ga_accounts)
       end
     end
   end
+
+  # Correct this to show the actual totals once we have correct passage of OSSE
+  # values all the way downstream.
+  def policy_responsible_amount(hbx_enrollment)
+    if hbx_enrollment.is_ivl_by_kind?
+      hbx_enrollment.total_premium - hbx_enrollment.applied_aptc_amount.to_f
```

Comment:
```
irb(main):001:0> a = 10.23
=> 10.23
irb(main):002:0> b = 11.67
=> 11.67
irb(main):003:0> c = b - a
=> 1.4399999999999995
```
this issue still exists for that reason in the glue we started using a module moneymath for float calculation 

```
module MoneyMath
  def as_dollars(val)
    BigDecimal.new(val.to_s).round(2)
  end
end
```